src/MonticelloRemoteRepositories/MCFtpRepository.class.st

```diff
@@ -123,11 +123,13 @@ MCFtpRepository >> user: userString [
 
 { #category : #required }
 MCFtpRepository >> writeStreamForFileNamed: aString replace: ignoreBoolean do: aBlock [
-	| stream |
-	stream := RWBinaryOrTextStream on: String new.
-	aBlock value: stream.
+
+	| bytes |
+	bytes := ByteArray streamContents: [ :stream |
```

Comment:
This can be `bytes := ByteArray streamContents: aBlock.`

Reply:
Thanks!